types/gapi.client.datacatalog-v1/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for non-npm package Google Cloud Data Catalog API v1 0.0
+// Project: https://cloud.google.com/data-catalog/docs/
+// Definitions by: Maxim Mazurok <https://github.com/Maxim-Mazurok>
+//                 Nick Amoscato <https://github.com/namoscato>
+//                 Declan Vong <https://github.com/declanvong>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
when you have time, please consider updating this, as it has no longer any value here on DT ecosystem, thanks!

Reply:
Will address in https://github.com/Maxim-Mazurok/google-api-typings-generator/issues/688 thanks!